os/include/tinyara/lcd/st7785.h

```diff
@@ -61,30 +65,30 @@
 #elif defined(CONFIG_LCD_RLANDSCAPE)
 #define LCD_ORIENTATION 0xa0
 #else
-#define LCD_ORIENTATION 0x70
+#define LCD_ORIENTATION 0xb0
 #endif
 
 #define LCDC_IMG_BUF_SIZE               LCDC_IMG_BUF_ALIGNED64B(LCD_XRES * LCD_YRES * 2)
 
-static const lcm_setting_table_t lcd_init_cmd_g[] = { 
+static const lcm_setting_table_t lcd_init_cmd_g[] = {
         {0x11, 0, {0x00}},
         {REGFLAG_DELAY, 120, {}},       /* Delayms (120) */
         {0x36, 2, {0x00, LCD_ORIENTATION}},
         {0x3A, 2, {0x00, 0x55}},// changed from 66 to 55 because 55 is for rgb565 and 66 is for rgb888
-        {0xB0, 2, {0x00, 0x10}},//changed 0x10 to 0x12
+        {0xB0, 2, {0x00, 0x10}},
         {0xB2, 10, {0x00, 0x0C, 0x00, 0x0C, 0x00, 0x00, 0x00, 0x33, 0x00, 0x33}},
-        {0xB7, 2, {0x00, 0x51}},
-        {0xBB, 2, {0x00, 0x22}},
+        {0xB7, 2, {0x00, 0x57}},
+        {0xBB, 2, {0x00, 0x1A}},
         {0xC0, 2, {0x00, 0x2C}},
         {0xC2, 2, {0x00, 0x01}},
-        {0xC3, 2, {0x00, 0x13}},
+        {0xC3, 2, {0x00, 0x0F}},
         {0xC6, 2, {0x00, 0x0F}},
         {0xD0, 2, {0x00, 0xA7}},
-        {0xD0, 4, {0x00, 0xA4, 0x00, 0xA1}}, //removed 0x00 and 0xA1
-        {0xE0, 28, {0x00, 0xF0, 0x00, 0x00, 0x00, 0x04, 0x00, 0x03, 0x00, 0x05, 0x00, 0x04, 0x00, 0x2E, 0x00, 0x44, 0x00, 0x45, 0x00, 0x39, 0x00, 0x14, 0x00, 0x14, 0x00, 0x2D, 0x00, 0x35}},
-        {0xE1, 28, {0x00, 0xF0, 0x00, 0x0A, 0x00, 0x0E, 0x00, 0x0F, 0x00, 0x0B, 0x00, 0x26, 0x00, 0x2E, 0x00, 0x43, 0x00, 0x45, 0x00, 0x36, 0x00, 0x12, 0x00, 0x12, 0x00, 0x2A, 0x00, 0x32}},
-        {0x2A, 6, {0x00, 0x00, 0x00, 0x00, LCD_CASET && 0xFF00, LCD_CASET && 0x00FF}},//for landscape it is 0x13F and portrait , it is 0xEF
-        {0x2B, 6, {0x00, 0x00, 0x00, 0x00, LCD_RASET && 0xFF00, LCD_RASET && 0x00FF}},//EF changed to 13F as given in datasheet
+        {0xD0, 4, {0x00, 0xA4, 0x00, 0xA1}},
+        {0xE0, 28, {0x00, 0xF0, 0x00, 0x01, 0x00, 0x08, 0x00, 0x05, 0x00, 0x07, 0x00, 0x15, 0x00, 0x32, 0x00, 0x44, 0x00, 0x49, 0x00, 0x37, 0x00, 0x12, 0x00, 0x11, 0x00, 0x2E, 0x00, 0x33}},
+        {0xE1, 28, {0x00, 0xF0, 0x00, 0x0C, 0x00, 0x10, 0x00, 0x0E, 0x00, 0x0C, 0x00, 0x06, 0x00, 0x32, 0x00, 0x33, 0x00, 0x49, 0x00, 0x35, 0x00, 0x10, 0x00, 0x12, 0x00, 0x2C, 0x00, 0x34}},
+        {0x2A, 8, {0x00, 0x00, 0x00, 0x00, 0x00, LCD_CASET1, 0x00, LCD_CASET2}},
+        {0x2B, 8, {0x00, 0x00, 0x00, 0x00, 0x00, LCD_RASET1, 0x00, LCD_RASET2}},
```

Comment:
As @edwakuwaku left a comment, how about adding comments to explain what each line means?